modules/logging-release-notes-5-8-16.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-16_{context}"]
+= Logging 5.8.16
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10989[RHBA-2024:10989] and link:https://access.redhat.com/errata/RHSA-2024:143685[RHBA-2024:143685].
+
+[id="logging-release-notes-5-8-16-bug-fixes_{context}"]
+== Bug fixes
+None.
```

Comment:
@xperimental do these Fixed Jira issues from the Errata need to go here as bug fixes?
[LOG-6322](https://issues.redhat.com/browse/LOG-6322) [release-5.8] Disable automatic discovery of log level in Loki for OpenShift Logging
[LOG-6233](https://issues.redhat.com/browse/LOG-6233) CVE-2024-47875 logging-view-plugin-container: nesting-based mutation XSS vulnerability [openshift-logging-5.8]

Reply:
Yes, LOG-6322 needs an entry in the release notes. LOG-6233 is a CVE, so afaik it goes into the other section.

Reply:
Ok yeaaaa I have 47875 in the CVE list 😅 ok cool. And added the bug fix. Hopefully this is squared away now 🤞